### PR TITLE
Receive pointer events through PointerPackets

### PR DESCRIPTION
**Pull request by jason-simmons**

### CODE REVIEW

File: packages/flutter/lib/src/gestures/events.dart [hunk 1]
```diff
@@ -72,4 +72,33 @@ class PointerInputEvent extends InputEvent {
...
+    result += "pressureMax: $pressureMax, ";
+    result += "distance: $distance, ";
+    result += "distanceMin: $distanceMin, ";
+    result += "distanceMax: $distanceMax, ";
+    result += "radiusMajor: $radiusMajor, ";
+    result += "radiusMinor: $radiusMinor, ";
+    result += "radiusMin: $radiusMin, ";
+    result += "radiusMax: $radiusMax, ";
+    result += "orientation: $orientation, ";
+    result += "tilt: $tilt)";
```
Comment (abarth) [inline packages/flutter/lib/src/gestures/events.dart]: You can get the compiler to do this string concatenation for you if you leave off the `;` and the `result += `.

```
+    String result = "PointerInputEvent("
+        "pointer: $pointer, "
+        "kind: $kind, "
+        "x: $x, "
+        "y: $y, "
```

File: packages/flutter/lib/src/rendering/binding.dart [hunk 2]
```diff
@@ -35,46 +39,42 @@ class BindingHitTestEntry extends HitTestEntry {
...
   static int _pointerCount = 0;
 
-  static PointerInputEvent convertPointerEvent(ui.PointerEvent event) {
-    Point position = new Point(event.x, event.y);
+  static List<PointerInputEvent> convertPointerPacket(PointerPacket packet) {
+    return packet.pointers.map(_convertPointer).toList();
+  }
 
-    _PointerState state = _stateForPointer[event.pointer];
+  static PointerInputEvent _convertPointer(Pointer ptr) {
```
Comment (abarth) [inline packages/flutter/lib/src/rendering/binding.dart]: s/ptr/pointer/

Please use complete words in variable names.

**Comment by abarth** [comment]
LGTM

File: packages/flutter/lib/src/gestures/events.dart (hunk 1 above)
Comment (jason-simmons) [inline packages/flutter/lib/src/gestures/events.dart]: done

File: packages/flutter/lib/src/rendering/binding.dart (hunk 2 above)
Comment (jason-simmons) [inline packages/flutter/lib/src/rendering/binding.dart]: done.  Although imho it's a bit confusing to have an object called "pointer" with a field called "pointer" (the pointer index)

Comment (abarth) [inline packages/flutter/lib/src/rendering/binding.dart]: Yeah, we should probably rename the filed to `id` or something.

**Pull request merged**